.travis.yml

```diff
@@ -1,7 +1,6 @@
 ---
 language: node_js
-node_js:
-  - "4"
+node_js: lts/carbon
```

Comment:
Carbon is Node 8, but we still support Node 6 as the minimum. Actually we even support Node 4 with the current release. If we drop support that would be a breaking change and we should adjust the support declaration in the `package.json` file.